controllers/comments-controllers.js

```diff
@@ -0,0 +1,8 @@
+const { selectCommentsByArticleId } = require("../models/comments-models");
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  selectCommentsByArticleId(article_id).then((comments) => {
+    res.status(200).send({ comments });
+  });
+};
```

Comment:
Really lovely code. Also good to see that you've not added error handling before the sad path tests.